README.md

```diff
@@ -13,9 +13,10 @@ $ yarn start
 
```

Comment:
I haven't managed to make it work without d2-ui link. Anyway let's try to merge this PR and we can review the error together during our next meeting...